stream-chat-android-ui-components/src/main/res/values-it/strings_message_input.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="stream_ui_message_input_send_to_channel">Also send to channel</string>
+    <string name="stream_ui_message_input_send_as_direct_message">Also send as direct message</string>
+    <string name="stream_ui_message_input_reply">Reply to message</string>
+    <string name="stream_ui_message_input_hint">Write something here</string>
+    <string name="stream_ui_message_input_only_attachments_hint">Add a comment or send</string>
+
+    <!-- Commands Popup -->
+    <string name="stream_ui_message_input_instant_commands">Instant Commands</string>
+    <string name="stream_ui_message_input_command_template">/%1$s %2$s</string>
+
+    <!-- File Picker Dialog -->
+    <string name="stream_ui_message_input_camera_access">Allow access to your Camera</string>
+    <string name="stream_ui_message_input_files_access">Allow access to your Files</string>
+    <string name="stream_ui_message_input_gallery_access">Allow access to your Gallery</string>
+    <string name="stream_ui_message_input_recent_files">Recent Files</string>
+    <string name="stream_ui_message_input_no_files">There is no photo or video.</string>
+    <string name="stream_ui_message_input_capture_media">Capture Image or Video</string>
+
+    <!-- System Permission Dialog -->
+    <string name="stream_ui_message_input_permission_camera_title">Camera and Storage Access Permission</string>
+    <string name="stream_ui_message_input_permission_camera_message">Camera and storage access permissions are needed in order to be able to take photos and videos</string>
+    <string name="stream_ui_message_input_permission_storage_title">Storage Access Permissions</string>
+    <string name="stream_ui_message_input_permission_storage_message">Storage access permissions is needed in order to be able to send and save photos, videos, music and the other files</string>
+    <string name="stream_ui_message_input_permission_setting_message">Enable permissions on App Settings</string>
+    <string name="stream_ui_message_input_permissions_setting_button">Settings</string>
+
+    <!-- Errors -->
+    <string name="stream_ui_message_input_error_file_size">File too big</string>
```

Comment:
`File troppo grande` this string might not fit in one row, you can wrap it in the second row. Also I would suggest to change the english text to `File too large`